qa/update-tests/src/test/java/io/camunda/zeebe/test/RollingUpdateTest.java

```diff
@@ -69,6 +71,10 @@ final class RollingUpdateTest {
           .withNetwork(network)
           .build();
 
+  @SuppressWarnings("unused")
+  @RegisterExtension
+  private final ContainerLogsDumper logsPrinter = new ContainerLogsDumper(cluster::getNodes);
```

Comment:
:question: Do we want this always? Out of curiosity or was that because of debugging?

Reply:
It's because if the test fails due to exporter state, and we don't print the logs on failure, it's very hard to diagnose the issue :smile: